### ISSUE TITLE
Issue with the newly introduced `fail_level`?

### ISSUE THREAD

**Issue by tagliala** [reviewdog/action-actionlint#149]
it appears that this input is not being detected

```
.github/workflows/github-actions-lint.yml:21:11: input "fail_level" is not defined in action "reviewdog/action-actionlint@v1". available inputs are "actionlint_flags", "fail_on_error", "filter_mode", "github_token", "level", "reporter", "reviewdog_flags", "tool_name" [action]
.github/workflows/rubocop.yml:39:11: input "fail_level" is not defined in action "reviewdog/action-rubocop@v2". available inputs are "fail_on_error", "filter_mode", "github_token", "level", "only_changed", "reporter", "reviewdog_flags", "rubocop_extensions", "rubocop_flags", "rubocop_version", "skip_install", "tool_name", "use_bundler", "workdir" [action]
reviewdog: found at least one issue with severity greater than or equal to the given level: any
```

I'm not familiar with this library, I don't know how to fix this

Also, I've tested against `main` with the same outcome

Ref: https://github.com/activeadmin/arbre/pull/648

**Comment by massongit** [comment]
This problem will be resolved when https://github.com/rhysd/actionlint/pull/487 is released and `reviewdog/action-actionlint` is made compatible with it.

**Comment by massongit** [comment]
@tagliala https://github.com/rhysd/actionlint/pull/487 was released as actionlint v1.7.5 and I released reviewdog/action-actionlint v1.61.0 which applies it.
Therefore, please retry running CI again with this applied.

**Comment by tagliala** [comment]
Thanks, CI passed as expecetd
